src/scipp/visualization/formatting_datagroup_html.py

```diff
@@ -86,6 +86,15 @@ def _summarize_atomic_variable(var: Any, name: str, depth: int = 0) -> str:
     if isinstance(var, Dataset | DataArray | Variable):
         parent_obj_str = "scipp"
         shape_repr = _format_shape(var)
+    if hasattr(var, '__module__'):
+        parent_obj_str = var.__module__.partition('.')[0]
+    if hasattr(var, 'sizes') and isinstance(var.sizes, Mapping):
+        try:
+            shape_repr = _format_shape(var)
+        except Exception:  # noqa: S110
+            # We don't want the rendering to fail because some
+            # object did not behave the way we expected here
+            pass
```

Comment:
Either you meant to use `elif` somewhere, or the branch for Scipp can be removed?